model/camera/config.go

```diff
@@ -35,30 +35,30 @@ type Config struct {
 }
 
 // Validate validates the configuration for the video storage camera component.
-func (cfg *Config) Validate(path string) ([]string, error) {
+func (cfg *Config) Validate(path string) ([]string, []string, error) {
```

Comment:
not super related to the PR, but what version of RDK makes this breaking change do you know?